From a5f2f97a5f77c33ee235e01844a5472416c391c3 Mon Sep 17 00:00:00 2001
From: flintman <flintman@flintmancomputers.com>
Date: Thu, 7 Nov 2013 19:50:06 -0500
Subject: [PATCH] build:  Add USE_SET_METADATA tag

On some of the older devices the recovery doesn't support
metadata when setting permissions.  This allows to use
set_perm vs set_metadata you will need to set
USE_SET_METADATA := false inside your boardconfig.mk

Change-Id: Ib6cf97b5ca28cee8a87975231f3a61b156e77d29
---
 core/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/Makefile b/core/Makefile
index 97cfb15..b8b4bee 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1328,7 +1328,9 @@ ifdef PRODUCT_EXTRA_RECOVERY_KEYS
 	$(hide) echo "extra_recovery_keys=$(PRODUCT_EXTRA_RECOVERY_KEYS)" >> $(zip_root)/META/misc_info.txt
 endif
 	$(hide) echo "mkbootimg_args=$(BOARD_MKBOOTIMG_ARGS)" >> $(zip_root)/META/misc_info.txt
+ifneq ($(USE_SET_METADATA),false)
 	$(hide) echo "use_set_metadata=1" >> $(zip_root)/META/misc_info.txt
+endif
 	$(call generate-userimage-prop-dictionary, $(zip_root)/META/misc_info.txt)
 	@# Zip everything up, preserving symlinks
 	$(hide) (cd $(zip_root) && zip -qry ../$(notdir $@) .)
-- 
1.8.4

